src/Login/containers/LoginContainer.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Login from '../components/LoginComponent';
+import { loginUtil } from '../redux/validations';
+import loginAction from '../redux/actions/LoginAction';
+
+
+export class LoginContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      errors: {}
+    };
+  }
+  onChange = (event) => {
+    this.setState({ [event.target.name]: event.target.value });
+  }
+
+  checkValidEmail = (userEmail) => {
+    const mailFormat = loginUtil.LOGIN_UTILS;
+    if (userEmail.match(mailFormat)) {
+      return true;
+    }
+    return false;
+  };
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    const {email, password} = this.state
+    const userdata = {
+      user: {
+        email,
+        password
+      }
+    };
+    
+    if (email === "" ) {
+      this.setState({ errors: { userEmail: 'Email field required' } })
+    } else if (!this.checkValidEmail(email)) {
+      this.setState({ errors: { userEmail: 'Invalid email format' } });
+    }else if (password ==="" ) {
+      this.setState({ errors: { userPassword: 'Password field is required'} });
+    } else if (password > 8) {
```

Comment:
Well this was supposed to check for the password passed in by the user if its greater than 8 but then l was told not to specify the message l display or l return to the user but l guess l forgot to change the operator but l have worked on it.

Reply:
So its checking for the password a user is passing in if its less than 8 .since in our backend we had implemented it to be at least  8 character. So l have worked on it well now.

Reply:
Isn't supposed to be `password.length` ?

Reply:
Actually this was removed since l do not need to validate for the password length since this is validated for as the user sign up and another thing l just provide a general message incaes they enter a wrong password